src/com/shashi/beans/ProductBean.java

```diff
@@ -14,6 +14,8 @@ public ProductBean() {
 	private String prodType;
 	private String prodInfo;
 	private double prodPrice;
+	private double discountPrice;
```

Comment:
I don't think we need discountPrice since @compuvad3 created a "[discountedPrice](https://github.com/matiasbeniluz/concordia-cart/blob/54dc48fd8bd8e533395782736faac71a11ab3e7f/src/com/shashi/beans/DiscountBean.java#L66)" method in DiscountBean. also if we hardcode the discountPrice here, it won't be updated when we update the discount